This patch make sure that the internal libmp4v2 is not built, and mpeg4ip is
used instead.
It also disables mpeg4ip plugin as version 1.4.1 is incompatible with it.
Index: faad2-2.0/configure.in
===================================================================
--- faad2-2.0.orig/configure.in
+++ faad2-2.0/configure.in
@@ -66,26 +66,8 @@ MY_CHECK_TYPEDEF_FROM_INCLUDE([fpos_t fo
         [#include <stdio.h>],
         [HAVE_FPOS_T_POS])
 
-
-AC_CHECK_PROG(external_mp4v2, mpeg4ip-config, yes, no)
-AC_CHECK_LIB(mp4v2, MP4Create, , external_mp4v2=no, -lstdc++)
-AC_CHECK_LIB(mp4v2, MP4MetadataDelete, , external_mp4v2=no, -lstdc++)
-AC_CHECK_HEADER(mp4.h, , external_mp4v2=no)
-
 AM_CONDITIONAL(WITH_MP4V2, false)
-
-if test x$external_mp4v2 = xyes; then
-  AM_CONDITIONAL(HAVE_MPEG4IP, true)
-  MPEG4IP_PLAYER_PLUGIN_DIR=`mpeg4ip-config --player-plugin-dir`  
-  AC_SUBST(MPEG4IP_PLAYER_PLUGIN_DIR)
-  AC_MSG_NOTICE([*** Building with external mp4v2 ***])
-else
-  AC_MSG_NOTICE([*** Building with internal mp4v2 ***])
-  AM_CONDITIONAL(HAVE_MPEG4IP, false)
-  if test x$WITHMP4V2 = xyes; then
-     AM_CONDITIONAL(WITH_MP4V2, true)
-  fi
-fi
+AM_CONDITIONAL(HAVE_MPEG4IP, false)
 
 if test x$WITHXMMS = xyes; then
   AC_CHECK_PROGS(XMMS_CONFIG, xmms-config,"not_found")
